### PR TITLE
doc updates, ips script updates

### PR DESCRIPTION
**Pull request by Rqnsom**
```
    doc: update test instructions

    - Allow to run tests with multiple threads
    - Separate and update instructions for running preformance tests
```
```
    chore: run preformance tests with a specific feature

    Performance test require additional configuration,
    so run those tests with:

       cargo +stable test performance --features performance
``` 
```
    chore: ips.py update

    - run only the necessary commands within the sudo context
    - print commands which the script runs - better output feedback
```

### CODE REVIEW

File: tools/ips.py [hunk 1]
```diff
@@ -36,31 +36,36 @@ def generate_hwaddr():
...
 
 
 def add_addr_to_existing_dev(device_name, ip_addr):
     if sys.platform.startswith('linux'):
-        cmd = 'ip addr add ' + ip_addr + ' dev ' + device_name + ' && '
+        cmd = 'sudo ip addr add ' + ip_addr + ' dev ' + device_name + ' && '
         cmd += 'ip link set ' + device_name + ' up'
     else:
-        cmd = 'ifconfig ' + device_name + ' alias ' + ip_addr + '/32'
+        cmd = 'sudo ifconfig ' + device_name + ' alias ' + ip_addr + '/32 &'
```
Comment (Rqnsom) [inline tools/ips.py:58]: cmd = 'sudo ifconfig ' + device_name + ' alias ' + ip_addr + '/32 &'

added & at the end to make script faster

File: README.md [hunk 2]
```diff
@@ -10,32 +10,51 @@ Ziggurat is written in stable Rust; you can install the Rust toolchain by follow
...
+ cargo +stable test
+```
+
+### Run performance tests
+Create a package of IP addresses (4k addresses) which are required for performance tests. From the root repository directory run:
+
+#### Preconditions under Linux
+Generate dummy devices with addresses:
+```zsh
+ python3 ./tools/ips.py --subnet 1.1.0.0/20 --file src/tools/ips.rs --dev_prefix test_zeth
```
Comment (Rqnsom) [inline README.md:41]: sudo context is moved within the script, so by default, all files created by the script are owned by the normal user

File: tools/ips.py [hunk 3]
```diff
@@ -36,31 +36,36 @@ def generate_hwaddr():
...
+    print(cmd)
     return os.system(cmd)
 
 
 def delete_addr_from_existing_dev(device_name, ip_addr):
     if sys.platform.startswith('linux'):
-        cmd = 'ip addr del ' + ip_addr + '/32 dev ' + device_name
+        cmd = 'sudo ip addr del ' + ip_addr + '/32 dev ' + device_name
     else:
         cmd = 'ifconfig ' + device_name + ' -alias ' + ip_addr
```
Comment (Rqnsom) [inline tools/ips.py:67]: missing sudo here...

File: README.md [hunk 4]
```diff
@@ -10,32 +10,51 @@ Ziggurat is written in stable Rust; you can install the Rust toolchain by follow
...
+
+### Run tests
+Run conformance and resistance tests with the following command:
+
+```zsh
+ cargo +stable test
+```
+
+### Run performance tests
+Create a package of IP addresses (4k addresses) which are required for performance tests. From the root repository directory run:
```
Comment (asmie) [inline README.md:36]: Maybe there should be some better ending - when I read it I'd expect some commands after "run:" or some short explanation and command, not the sub-section. 
When you splitted Linux and Mac under two sections I suggest to change it to:
`From the root repository directory, depending on your OS, run one of the following commands.`

File: README.md (hunk 2 above)
Comment (asmie) [inline README.md:41]: If we remove `sudo` call from here and use it somewhere in the script, there should be notification here, that user must be in sudoers file to use this script.
Previous version of this script was privilages agnostic - it assumed that was run under root privilages (some Linux distro has no `sudo` by default and therefore user can use `su` to become root and run the script as usual - as user can see `sudo` in script invocation).

**Review by asmie — APPROVED** [review]
LGTM - just a little need to update doc to provide user with info about sudoers file.

File: README.md (hunk 2 above)
Comment (Rqnsom) [reply README.md:41]: Let me try to explain it in words in the form of a note before the section split (mac/Linux)

File: tools/ips.py (hunk 3 above)
Comment (Rqnsom) [reply tools/ips.py]: Done

File: README.md (hunk 4 above)
Comment (Rqnsom) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (Rqnsom) [reply README.md:46]: Done

**Pull request merged**